src/daisy.js

```diff
@@ -132,12 +149,60 @@ var daisy = (function(Object, Array, Math, undefined){
 	var Computation = function(initParam, options){
 		this.currentVal = initParam + '';
 		this.options = options || {};
+		if (this.options.round && !this.options.format) {
+			this.options.format = 'n';
+		}
+		if (this.options.INR && this.options.commas === void 0) {
+			this.options.commas = true;
+		}
+	};
+
+	Computation.prototype.format = function () {

```

Comment:
I'm not too crazy about this function being exposed as part of the prototype.  After all, daisy returns a ```Computation``` instance, so ```format``` will be available as an exposed function as written.  Can we make this private and just pass in the ```options``` object (like what we are doing with ```add```, ```divide```, etc?